xgcm/test/test_grid.py

```diff
@@ -783,13 +795,22 @@ def test_keep_coords(funcname, gridtype):
         result = func(ds.tracer, axis_name)
         base_coords = list(result.dims)
         augmented_coords = [
-            c for c in ds.tracer.coords if set(ds[c].dims).issubset(result.dims)
+            c
```

Comment:
Need to revert this too, please disregard for now.

Reply:
I take that back. I believe the old test was not properly testing the behavior we want to ensure here. I hope this small change to tests is ok.